napari/_qt/qt_main_window.py

```diff
@@ -1550,27 +1555,62 @@ def _screenshot(
         flash : bool
             Flag to indicate whether flash animation should be shown after
             the screenshot was captured.
-        size : tuple (int, int)
+        size : Optional[int, int]
```

Comment:
```suggestion
        size : tuple of two ints, optional
```
https://numpydoc.readthedocs.io/en/latest/format.html#parameters